sql/2020/16_Ecommerce/pagestats_image_bydevice.sql

```diff
@@ -7,20 +7,19 @@ SELECT
   ROUND(APPROX_QUANTILES(bytesImg, 1000)[OFFSET(percentile * 10)] / 1024, 2) AS image_kbytes
 FROM
   `httparchive.summary_pages.2020_08_01_*`
-JOIN
-  `httparchive.technologies.2020_08_01_*`
+JOIN (
+  SELECT  DISTINCT 
+    _TABLE_SUFFIX,
+    url
+  FROM `httparchive.technologies.2020_08_01_*`
+  WHERE category = 'Ecommerce')
 USING (_TABLE_SUFFIX, url),
 UNNEST([10, 25, 50, 75, 90]) AS percentile
-WHERE
-  category = 'Ecommerce' AND 
-  (app != 'Cart Functionality' AND 
-   app != 'Google Analytics Enhanced eCommerce')
 GROUP BY
   percentile,
   client
-ORDER BY
-  percentile,
-  order
 ORDER BY
   percentile,
   client
+  
+
```

Comment:
Not, remove extra newline:

```suggestion

```

I presume these are the results that were previously saved and nothing needs to be updated in the figures?

Reply:
Yes... results sheet and chapter is already using data from this query.